Slim/Plugin/Podcast/Search.pm

```diff
@@ -0,0 +1,111 @@
+package Slim::Plugin::Podcast::Search;
+
+# Logitech Media Server Copyright 2005-2020 Logitech.
+
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License, 
+# version 2.
+
+use strict;
+
+use JSON::XS::VersionOneAndTwo;
+use XML::Simple;
+
+use Slim::Utils::Cache;
+use Slim::Utils::Log;
+use Slim::Utils::Prefs;
+use Slim::Utils::Strings qw(string cstring);
+
+my $log   = logger('plugin.podcast');
+my $prefs = preferences('plugin.podcast');
+
+my @providers = ( {
+		name   => 'Apple/iTunes',
+		result => 'results',
+		feed   => 'feedUrl',
+		title  => 'collectionName',
+		image  => ['artworkUrl600', 'artworkUrl100'],
+		setup => sub {
+			my $url = 'https://itunes.apple.com/search?media=podcast&term=' . shift;
+			my $country = $prefs->get('country');			
+			return $url .= "&country=$country" if $country;
```

Comment:
Oh, and that would not return the `$url` if no country was set.

Reply:
But I wanted to play nice with users because the API works w/o a country